components/br-stream/src/metadata/store/slash_etc.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+use std::{
+    cell::Cell,
+    collections::{BTreeMap, HashMap},
+    ops::Bound,
+    sync::Arc,
+};
+
+use crate::{
+    errors::Result,
+    metadata::{
+        keys::{KeyValue, MetaKey},
+        store::{
+            GetResponse, KvChangeSubscription, KvEvent, KvEventType, MetaStore, Snapshot,
+            Subscription, WithRevision,
+        },
+    },
+};
+use async_trait::async_trait;
+use tokio::sync::{
+    mpsc::{self, Sender},
+    Mutex,
+};
+use tokio_stream::StreamExt;
+struct Subscriber {
```

Comment:
the vertical spacing here is strange, there should be no newline on line 8 and there should be a new line before line 25.